providers/maqsam/jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+};
```

Comment:
Delete this file from the PR. It is not needed as we pasted the config inside of package.json and having it doesn't apply the changes needed to execute Axios in Jest because it takes precedence.

Reply:
Got it on my way